packages/amplify-graphql-api-construct/API.md

```diff
@@ -119,8 +119,9 @@ export interface AmplifyGraphqlApiResources {
 
 // @public
 export class AmplifyGraphqlDefinition {
+    static fromBindingAndFiles(modelDataSourceBinding: ModelDataSourceBinding, ...filePaths: string[]): IAmplifyGraphqlDefinition;
     static fromFiles(...filePaths: string[]): IAmplifyGraphqlDefinition;
```

Comment:
I think if we don't just have a pure vararg I'd avoid the spread on filePaths, and explicitly pass in an array.

Reply:
Went back & forth on this. I generally disfavor varargs in any case since it makes extensions hard (which is why we can't just add an argument to the `fromFiles` static factory method. I'll rework the signature to `fromFilesAndBinding(filePaths: string[], modelDataSourceBinding: ModelDataSourceBinding)` to keep it parallel with `fromString` above.

Reply:
Yeah, I have pretty immediate regrets on making that a varargs in the first place, since we can't easily change it now w/o impact to callers.